test/test_queue.js

```diff
@@ -977,6 +977,48 @@ describe('Queue', function () {
         }).catch(done);
       });
     });
+
+    it('should pause the queue locally when more than one worker is active', function () {
+      var queue1 = utils.buildQueue('pause-queue');
+      var queue1IsProcessing = new Promise(function(resolve, reject) { //eslint-disable-line no-unused-vars
```

Comment:
Just habit. `new Promise(function(resolve){})` always looks odd to me. I'll remove them.